pkgs/development/libraries/lirc/default.nix

```diff
@@ -62,6 +62,10 @@ stdenv.mkDerivation rec {
   preConfigure = ''
     # use empty inc file instead of a from linux kernel generated one
     touch lib/lirc/input_map.inc
+
+    # The PKG_CONFIG= option doesn't work, and neither does the environment variable.
+    substituteInPlace ./configure \
```

Comment:
could you please set PKGCONFIG instead patching configure, similiar to https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/talloc/default.nix#L43

This pattern is more common than patching.